sdks/go/test/integration/integration.go

```diff
@@ -88,13 +88,17 @@ var flinkFilters = []string{
 }
 
 var samzaFilters = []string{
+	// XLang tests no supported by samza runner.
+	"TestXLang.*",
```

Comment:
Can we create a ticket ?

Reply:
In addition, judging from the log, it seems only `TestXLang_CoGroupBy` is failing but other `TestXLang` are succeeding, can we exclude `TestXLang_CoGroupBy` only?